.gitignore

```diff
@@ -27,3 +27,6 @@ android/app/local.properties
 
 # Debug Logs
 debug.log
+
+# Generated documentation
+docs/
```

Comment:
well, till now we werent' publishing the docs directly in github - rather than leave it up to the user to check them out by running the `doc` script

Reply:
... which would mean we would eventually have two different directories, with slightly different names, as document folders, causing some confusion (or a lot)
I know we were not publishing the docs so far. But the issue here is what these generated documents are for: the use case is the local developer that will run the script themselves, or do we want to publish them on the web for everyone to see and make the repo more presentable? I think it's 2 what we want, and that implies two decisions: use the docs directory (for this and the architecture documents) *and* not ignore it. Since actually changing the repo configuration it's something you and I cannot do, I think that not adding it to the .gitignore file clearly signals the intent, if that's precisely where we are going.

Reply:
(That would, BTW, also remove the test fail that's there)

Reply:
I was thinking the same... to add a `docs/documentation` folder where all generated docs are located (and can be ignored instead of uploading the git) and rest can lie in `docs/` as it is now..

Reply:
As long as it's not ignored, I'm good with it. If it's ignored, we would have to delete that later on. But if you think the first option (dev-generated docs) is what's going to win in the end, this is a nice tradeoff, because just removing that entry could change to option 2.